src/test/acceptance/features/navigation.feature

```diff
@@ -30,10 +30,10 @@ Feature: Application process navigation is controlled
     Then I am shown the overview page
     Examples:
       | page                     |
-      | enter name               |
-      | enter national insurance |
       | enter date of birth      |
       | are you pregnant         |
+      | enter name               |
+      | enter national insurance |
       | card address             |
 
   Scenario Outline: Navigation is not allowed past the current page in the flow
```

Comment:
I feel like the examples (needs expanding to view code) in this feature should not work as they have not been updated, but nothing is failing. Anyone got any thoughts on this?

Reply:
This feature doesn't care about the order of the pages, just the state of the application, i.e IN_REVIEW or COMPLETE.